test/node/conversation-group.js

```diff
@@ -77,10 +76,35 @@ describe('Group Conversation', () => {
 
     it('should get the participants of the conversation', async () => {
         const res = await client.getConversationParticipants(conversation.convId);
-        res.participants.forEach(participant => {
-            if (!(participant.userId === user.userId || participant.userId === user2.userId)) {
-                assert(false);
-            }
-        });
+        assert(res && res.participants.some(u => u.userId === user.userId) && res.participants.some(u => u.userId === user2.userId));
+    });
+
+    it('should update the conversation and raise a conversationUpdated event', async () => {
+        const topic = `${Date.now()}z`;
+        const data = {
+            topic: topic
+        }
+        const res = await Promise.all([
+            client.updateConversation(conversation.convId, data),
+            helper.expectEvents(client, [{
+                type: 'conversationUpdated',
+                predicate: evt => evt.conversation.convId === conversation.convId && evt.conversation.topic === topic
+            }])
+        ]);
+        const convId = conversation.convId;
+        conversation = res[0];
+        assert(conversation.convId === convId && conversation.topic === topic);
+    });
+    // Requires permissions
```

Comment:
Add blank line above and change comment to: "Requires MODERATION permission"

Reply:
done